src/components/pages/courses/instructor-profile.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from 'next/link'
+import * as React from 'react'
+import Markdown from 'react-markdown'
+
+const InstructorProfile: React.FunctionComponent<{
```

Comment:
originally just started breaking out the shared components and then did the big one above. these ar5e still nicer here...